lib/aws/rails/sqs_active_job/executor.rb

```diff
@@ -16,13 +16,18 @@ class Executor
            fallback_policy: :caller_runs # slow down the producer thread
         }.freeze
 
-        def initialize(options = {})
+        def initialize(options = {}, refresh_timeout = nil)
```

Comment:
Why not include the refresh_timeout in the options?  This leads to a bit of a clunky interface when setting both.

Reply:
Was leaving it out since options is merged straight into defaults and passed to ThreadPoolExecutor. Happy to move it if you prefer combined.

Reply:
You can delete it out

Reply:
Yeah, +1 to doing something like `@refresh_timeout = option.delete(:refresh_timeout)`

Reply:
Done